src/applications/static-pages/additional-info-toggle.js

```diff
@@ -0,0 +1,18 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const expanderButton = document.getElementById('crisis-expander-link');
+
+  if (expanderButton) {
+    const expanderContent = document.getElementById('crisis-expander-content');
+    const chevron = expanderButton.querySelector('i.fa-angle-down');
+
+    expanderButton.addEventListener('click', () => {
+      const isAriaExpanded = JSON.parse(
+        expanderButton.getAttribute('aria-expanded'),
+      );
+
+      expanderButton.setAttribute('aria-expanded', `${!isAriaExpanded}`);
+      expanderContent.classList.toggle('expander-content-closed');
+      chevron.classList.toggle('open');
+    });
+  }
+});
```

Comment:
heavily based this function on `src/applications/static-pages/createAdditionalInfoWidget.js`

Reply:
Awesome find! I didn't know we had that already. This makes me wonder if we can share code. I messed with this locally using `pages/family-member-benefits.md` by replacing the alert banner on that page with the HTML below. This HTML follows what that `src/applications/static-pages/createAdditionalInfoWidget.js` expects.

```html
<div class="usa-alert usa-alert-warning">
  <div class="usa-alert-body">
    <h4 class="usa-alert-heading">How do I get help for a Veteran who's in crisis?</h4>
    <div class="form-expanding-group additional-info-container">
    <span class="additional-info-title">Find out how to get support anytime, day or night</span>
      <span>
        <div class="additional-info-content usa-alert-text">
          <p>If you're concerned about a Veteran in crisis, connect with our caring, qualified Veterans Crisis Line responders for confidential help. Many of them are Veterans themselves. This service is private, free, and available 24/7.</p>
        <p><strong>To connect with a Veterans Crisis Line responder anytime, day or night:</strong></p>
        <ul>
          <li>Call <a href="tel:+1-800-273-8255">1-800-273-8255</a>, then press 1.</li>
          <li><a href="https://www.veteranscrisisline.net/ChatTermsOfService.aspx?account=Veterans%20Chat/" class="no-external-icon">Start a confidential Veterans Chat</a>.</li>
          <li>Text <a href="sms:838255">838255</a>.</li>
          <li>If you have hearing loss, call TTY: 1-800-799-4889.</li>
        </ul>
        <p><strong>If you're concerned about a Veteran who's homeless or at risk of becoming homeless</strong></p>
	      <p>Call the National Call Center for Homeless Veterans at 1-877-4AID-VET (<a href="tel:+18774243838">1-877-424-3838</a>) for help 24 hours a day, 7 days a week. You or the Veteran can talk privately with a trained VA counselor for free.</p>
        </div>
      </span>
    </div>
  </div>
</div>
```

The toggle does work, except that the expanded content has an awkward blue border on the left side -

![image](https://user-images.githubusercontent.com/1915775/55825186-75a72b80-5ad3-11e9-9d77-11c416f96358.png)

I'm wondering if we can add a SCSS rule for the additional-info widgets inside of an alert banner to get rid of this banner. This rule could be a design system update, but for now, `src/applications/static-pages/sass/static-pages.scss` might be a good place for this rule to live. What do you think of this approach?

Reply:
any way to consolidate things further sounds like an improvement to me. I'll mess around with your approach and see how it feels refining it

Reply:
Thanks @hinzed1127!

Reply:
so after a little digging, it looks like those styles are coming from [formation](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/1556312a349c2a1aba3da24aa8d896287c4cb024/packages/formation/sass/modules/_m-form-elements.scss#L177)

Idk conventions for how you usually override those, but I could tack on a class and add more specifically scoped styles inside that `static-pages.scss` file?

Reply:
That is a great point. I think you have the right idea with putting the style in `static-pages.scss`., but in a follow-up PR we'll move the style into the design system. I think these changes makes sense to be there, since this is part of standardizing types of alerts.